apps/storybook/stories/Namekit/RegistrationPriceInfo.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  PrimaryRegistrationStatus,
+  SecondaryRegistrationStatus,
+  buildTimestamp,
+  now,
+} from "@namehash/ens-utils";
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  RegistrationPriceInfo,
+  RegistrationPriceInfoDisplayingFormat,
+} from "@namehash/namekit-react";
+
+const ELEVEN_DAYS_FROM_NOW_TIMESTAMP = buildTimestamp(
```

Comment:
I remember leaving a comment about this in another PR. Can you please look for where those comments were and follow up on all of those again in this PR?

I'm worried that some of the detailed comments and feedback I've shared somewhere else is getting lost.